transform/transformers/mbs_transformer.py

```diff
@@ -38,6 +38,15 @@ def round_mbs(value):
         """MBS rounding is done on a ROUND_HALF_UP basis and values are divided by 1000 for the pck"""
         return Decimal(round(Decimal(float(value))) / 1000).quantize(1)
 
+    @staticmethod
+    def convert_str(value):
+        """Convert submitted data to int in the transform"""
+        try:
+            return int(value)
+        except TypeError:
+            logger.info("Tried to transform None to int. Returning None.")
+            return None
```

Comment:
Should we checking for `None`? As it is if we got anything other than an integer in these fields we would drop the data.

Reply:
None is parsed from the dict in a later step

Reply:
But should we fail the transform rather than drop data?

Reply:
I will have to check with the business analysts 👍

Reply:
Spoke to @LJBabbage - we shouldn't fail the transform in this instance, instead drop the data (which is persisted in the DB in any case), and continue processing.